### PR TITLE
Replace %USER% in lock history messages

### PR DESCRIPTION
**Pull request by MarcusP-P**
This replaces the %USER% with either the appropriate user's name, or the extension name in lock history.

I'm not sure if this is the best way to do this.

### CODE REVIEW

File: app/components/LockHistory.jsx [hunk 1]
```diff
@@ -9,12 +9,21 @@ import GetLockHistory from '../graphql/GetLockHistoryQuery.graphql';
...
+  return e.user?.username || 'unknown';
+}
+
 const HistoryList = memo(({ history }) => {
   const itemContent = useCallback((i, e) => (
     <>
       <Accordion>
         <AccordionSummary expandIcon={<ExpandMore/>}>
-          <Typography flexGrow={1}>{e.title}</Typography>
+          <Typography flexGrow={1}>{e.title.replace('%USER%', historyUserName(e))}</Typography>
```
Comment (KittenApps) [inline app/components/LockHistory.jsx]:
```suggestion
          <Typography flexGrow={1}>{e.title.replace('%USER%', e.role === 'extension' ? e.extension : e.user?.username || 'unknown')}</Typography>
```

**Review by KittenApps — CHANGES_REQUESTED** [review]
That's not the correct way to get the displayName of an extension. Showing the slug should do for now, until we get a proper extensions api in maybe 10 years hopefully.

**Comment by MarcusP-P** [comment]
I've added your patch, and removed the function. 

I assume the correct way of doing it would be to get the descriptions and slugs from the extensions endpoint and then look it up. That's a little beyond my React and GraphQL knowledge :)

**Comment by chrmoritz** [comment]
and the current API for that is total garbage, because it doesn't scale either server nor client side with all these custom extension getting registered. So better wait until a proper solutions is developed eventually.

Thank you again.

**Review by chrmoritz — APPROVED** [review]

**Review by KittenApps — APPROVED** [review]

**Pull request merged**